tempesta_fw/sock.c

```diff
@@ -1136,12 +1136,12 @@ ss_sock_create(int family, int type, int protocol, struct sock **res)
 	struct sock *sk = NULL;
 	const struct net_proto_family *pf;
 
-	rcu_read_lock();
+	rcu_read_lock_bh();
```

Comment:
Don't quite understand this replacement: it seems there are no spinlocks here (with possibility of user/softirq deadlock) that it was necessary to disable softirq; besides, `synchronize_rcu()` is used kernel with `net_families` array - not `*_bh` flavor.

Reply:
Basically the reason is the same as for https://github.com/tempesta-tech/tempesta/pull/916#discussion_r171842284 : `ss_sock_create()` can be called from process contest on the system start as well as from softirq context on server reconnection attempt (timer context). Updated https://github.com/tempesta-tech/tempesta/issues/736#issuecomment-369945612